shop3/looping-tasks/exts-common.lisp

```diff
@@ -0,0 +1,89 @@
+(in-package :shop)
+
+(defclass looping-domain (domain)
+     ()
+  (:documentation "Mixin domain class for HTNs with LOOP tasks in them."))
+
+
+(defmethod seek-plans-task ((domain looping-domain) task1 state tasks
```

Comment:
Could this be replaced by a simpler method that checks for `(looping-p (get-task-name task1))` and then calls `seek-plans-loop` or if that test fails, simply `call-next-method`?

By squashing all of the `seek-plans-task` code for other classes and mixins, this potentially breaks present and future code for other SHOP extensions.